bigbluebutton-html5/imports/ui/components/breakout-join-confirmation/component.jsx

```diff
@@ -20,7 +20,7 @@ const intlMessages = defineMessages({
     description: 'Join breakout confim message',
   },
   confirmLabel: {
-    id: 'app.breakoutJoinConfirmation.confirmLabel',
+    id: 'app.createBreakoutRoom.join',
```

Comment:
What happens now with `app.breakoutJoinConfirmation.confirmLabel` - do we remove it from en.json? This was the only place of use.

Reply:
Yeah, I removed it.